sormas-backend/src/test/java/de/symeda/sormas/backend/importexport/DatabaseExportServiceTest.java

```diff
@@ -78,7 +82,21 @@ public void test_all_entities_have_export_configuration() {
 				if (!exportableTables.contains(tableName)) {
 					missingEntities.add(clazz.getSimpleName());
 				} else if (NOT_EXPORTED_ENTITIES.contains(clazz.reflect())) {
-					exportedButNotWanted.add(clazz.getSimpleName());
+					exportedButNotWantedEntity.add(clazz.getSimpleName());
+				}
+
+				for (JavaMethod method : clazz.getMethods()) {
+					if (method.isAnnotatedWith(JoinTable.class)) {
+						JoinTable joinTableAnnotation = method.getAnnotationOfType(JoinTable.class);
+						String joinTableName = joinTableAnnotation.name();
+						if (StringUtils.isBlank(joinTableName)) {
+							joinTableName = clazz.getSimpleName().toLowerCase();
```

Comment:
the name of the join table should not be the name of the class.  Pleas let the test fail if there is no joinTableName